doc/lua_api.txt

```diff
@@ -6946,13 +6946,17 @@ object you are working with still exists.
             (default: `true`)
         * `texture`: A regular texture for the sun. Setting to `""`
             will re-enable the mesh sun. (default: "sun.png", if it exists)
+            At sunrise, the texture appears non-rotated on the screen if one
+            looks at it.
         * `tonemap`: A 512x1 texture containing the tonemap for the sun
             (default: `"sun_tonemap.png"`)
         * `sunrise`: A regular texture for the sunrise texture.
             (default: `"sunrisebg.png"`)
         * `sunrise_visible`: Boolean for whether the sunrise texture is visible.
             (default: `true`)
         * `scale`: Float controlling the overall size of the sun. (default: `1`)
+            Note: For legacy reasons, the sun is bigger than the moon by a factor
+            of `0.119/0.076` for equal `scale` values.
```

Comment:
Change ratio to '1.57', because it's simpler and more informative.

Reply:
I've written it like this so that one can find the values in the src by grepping.

Reply:
Done.
I've also made comments to the src to allow grepping for the magic numbers.